commons/src/main/java/io/hyscale/commons/utils/OperatingSystemUtil.java

```diff
@@ -16,13 +16,44 @@
 package io.hyscale.commons.utils;
 
 import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import io.hyscale.commons.config.SetupConfig;
 import io.hyscale.commons.constants.ToolConstants;
 
-public class OperatingSystemHelper {
+public class OperatingSystemUtil {
 	
+	private static final Logger logger = LoggerFactory.getLogger(OperatingSystemUtil.class);
+	public static final String HYSCALE_HOST_FS = System.getenv(ToolConstants.HYSCALE_HOST_FS_PROPERTY);
 	private static final String WINDOWS_FS_MATCHER = "\\\\";
 
+	/**
+	 * 
+	 * @return true if HYSCALE_HOST_FS property is that of windows
+	 */
+	public static boolean isHostWindows() {
+		if (WINDOWS_FS_MATCHER.equals(HYSCALE_HOST_FS)) {
+			logger.debug("The Host operating system is windows");
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * Update filepath to host file separator if the field is set
+	 * 
+	 */
+	public static String setWindowsFileSeparator(String filepath) {
```

Comment:
updateToHostFileSeparator() ???
it does not set the file separator directly check for windows and then sets right should be update